components/tag-list/test/tag-list.visual-diff.js

```diff
@@ -112,33 +125,28 @@ describe('d2l-tag-list', () => {
 	});
 
 	describe('clearable behavior', () => {
-		const selector = '#clearable';
 
-		before(async() => {
-			await page.$eval(selector, async(elem) => {
-				elem.parentNode.style.width = '1200px';
-				elem._showHiddenTags = false;
-				elem._hasShownKeyboardTooltip = true;
-				await elem.updateComplete;
-			});
-			await page.waitForTimeout(2000);
+		beforeEach(async() => {
+			await page.reload();
```

Comment:
There isn't really a great way to do this without a reload. Another option would be to add a new item before each tests get run for them to delete, but then that seems like it could have a bit of risk of failing silently (the before case would be the same as the after)

Reply:
If we're reloading before each, is there still a benefit of adding `#clearable2`?

Reply:
It's overall not needed, just separates the tests, which would be useful if it was switched to the second option.